packages/itwin/tree-widget/src/components/trees/models-tree/ModelsTreeComponent.tsx

```diff
@@ -151,11 +151,11 @@ function ModelsTreeComponentImpl(props: ModelTreeComponentProps & { iModel: IMod
         {props.headerButtons
           ? props.headerButtons.map((btn, index) => <Fragment key={index}>{btn({ viewport, models: availableModels })}</Fragment>)
           : [
-              <ShowAllButton viewport={viewport} models={availableModels} key="show-all-btn" density={props.density} />,
-              <HideAllButton viewport={viewport} models={availableModels} key="hide-all-btn" density={props.density} />,
-              <InvertButton viewport={viewport} models={availableModels} key="invert-all-btn" density={props.density} />,
-              <View2DButton viewport={viewport} models={availableModels} key="view-2d-btn" density={props.density} />,
-              <View3DButton viewport={viewport} models={availableModels} key="view-3d-btn" density={props.density} />,
+              <ShowAllButton viewport={viewport} models={availableModels} key="show-all-btn" density={props.density} onFeatureUsed={props.onFeatureUsed} />,
```

Comment:
`density` and `onFeatureUsed` should be also passed to custom button renderers

Reply:
Passed to custom renderers in `ModelsTreeComponenet` and `CategoriesTreeComponent`.